library/sys.po

```diff
@@ -2942,51 +3032,59 @@ msgstr ""
 
 #: library/sys.rst:1578
 msgid "Handle an unraisable exception."
-msgstr ""
+msgstr "Gère une exception *non levable* (**unraisable** en anglais)"
 
 #: library/sys.rst:1580
 msgid ""
 "Called when an exception has occurred but there is no way for Python to "
 "handle it. For example, when a destructor raises an exception or during "
 "garbage collection (:func:`gc.collect`)."
 msgstr ""
+"Appelé lorsqu'une exception s'est produite mais qu'il n'y a aucun moyen pour "
+"Python de la gérer. Par exemple, lorsqu'un destructeur lève une exception ou "
+"durant la **garbage collection** (:func:`gc.collect`)."
 
 #: library/sys.rst:1584
 msgid "The *unraisable* argument has the following attributes:"
 msgstr "Les arguments *unraisable* ont la signification suivante :"
 
 #: library/sys.rst:1586
 msgid "*exc_type*: Exception type."
-msgstr "*exc_type* : le type de l'exception ;"
+msgstr "*exc_type* : Type de l'exception."
 
 #: library/sys.rst:1587
 msgid "*exc_value*: Exception value, can be ``None``."
-msgstr "*exc_value*: la valeur de l'exception, peut être ``None`` ;"
+msgstr "*exc_value*: Valeur de l'exception, peut être ``None``."
 
 #: library/sys.rst:1588
 msgid "*exc_traceback*: Exception traceback, can be ``None``."
 msgstr ""
-"*exc_traceback* : la pile d'appels pour cette exception, peut être ``None`` ;"
+"*exc_traceback* : la pile d'appels pour cette exception, peut être ``None``."
 
 #: library/sys.rst:1589
 msgid "*err_msg*: Error message, can be ``None``."
-msgstr ""
+msgstr "*err_msg* : Message d'erreur, peut être ``None``."
 
 #: library/sys.rst:1590
 msgid "*object*: Object causing the exception, can be ``None``."
-msgstr ""
+msgstr "*object* : Objet causant l'exception, peut être ``None``."
 
 #: library/sys.rst:1592
 msgid ""
 "The default hook formats *err_msg* and *object* as: ``f'{err_msg}: {object!"
 "r}'``; use \"Exception ignored in\" error message if *err_msg* is ``None``."
 msgstr ""
+"Le point d'entrée par défaut formate *err_msg* et *object* de la façon "
+"suivante : ``f'{err_msg}: {object!r}'``; Utilisez \"*Exception ignored in*\" "
```

Comment:
Hum non je pense qu'ici c'est le truc qui formatte qui utilise "Exception ignored in", on ne demande pas à l'utilisateur de le faire.

Reply:
```suggestion
"suivante : ``f'{err_msg}: {object!r}'``; sinon utilise \"*Exception ignored in*\" "
```